librz/core/core.c

```diff
@@ -955,10 +957,39 @@ RZ_API int rz_core_fgets(char *buf, int len, void *user) {
 	return rz_str_ncpy(buf, ptr, len - 1);
 }
 
+static void append_marks(RzStrBuf *sb, RzList /*<RzMarkItem *>*/ *list, ut64 addr, bool is_start) {
+	RzListIter *it;
+	RzMarkItem *bi;
+	bool first = (rz_strbuf_length(sb) == 0);
+
+	rz_list_foreach (list, it, bi) {
+		if ((is_start && addr == bi->from) || (!is_start && addr == bi->to)) {
+			if (!first)
+				rz_strbuf_append(sb, "  ; ");
+			rz_strbuf_appendf(sb, is_start ? "[s] %s (0x%" PFMT64x "-0x%" PFMT64x ")" : "[e] %s (0x%" PFMT64x "-0x%" PFMT64x ")",
```

Comment:
```suggestion
			rz_strbuf_appendf(sb, is_start ? "[s] %s [0x%" PFMT64x "-0x%" PFMT64x "]" : "[e] %s [0x%" PFMT64x "-0x%" PFMT64x "]",
```
Or is the range exclusive?